src/templates/careerPage/careerPage.module.scss

```diff
@@ -23,14 +28,63 @@
   }
 
   .careerPerks {
-    display: flex;
-    flex-flow: row wrap;
-    justify-content: space-between;
+    div.circleGrid {
+      display: grid;
+      justify-items: center;
+      align-items: center;
+      grid-template-columns: 75px 350px 350px 350px 75px;
+      grid-template-rows: repeat(5, 195px);
+      margin-top: 7em;
+      margin-bottom: 5em;
 
-    section:nth-of-type(1) {
-    }
+      section:nth-of-type(1) {
```

Comment:
Her må vi finne på noe lurt så vi kan enkelt ta inn og ut perks

Reply:
Alle kommer fra CMSet

Reply:
Men hvis det kommer en ny perk i CMS, vil ikke stylingen brekke da ?

Reply:
Kan legge inn noe logikk  som setter fargeklasser. Vi kan lage f.eks et array av farger som vi itererer over og setter klasser på en og en perk ?

Reply:
Er ikke fargene jeg tenker på, men antallet perks. Sånn det er nå kan man redigere alt innholdet i de som er der, men man kan ikke legge til nye, fordi det kun er plass til 7 i den sirkelen. 

Fargene ligger i "section:nth-of-type(1-7)", så den første section som blir lista ut vil alltid få samme farge, samme hva innholdet er. Hvis du prøver å bytte fra /no/career til /en/career så ser du at perksene flytter seg rundt (de blir ikke listet ut i samme rekkefølge), men fargene forblir på samme sted.

Reply:
Ja jeg skjønner. Jeg mener at vi må sette det opp slik at vi kan fjerne og legge til perks akkurat som vi vil. Så må man naturlig nok ta ansvar selv og prøve å få det symetrisk. Men om man legger til en perk til så vil den legge seg nede i venstre hjørne, neste i høyre hjørnet osv...

Vi kan ikke la oss stoppe av CSS i å legge til eller fjerne perks. Derfor mente jeg det kunne være lurt å lage et array av farger og sette farger på sirklene tilfeldig i jS koden. På den måten har vi ingen begrensinger på antallet.